code/src/Appsinglepage.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'
+import 'App.css'
+
+export const App = () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('')
+  const [number, setNumber] = useState('');
+  const [age, setAge] = useState();
+  const [showSummary, setShowSummary] = useState (false);
```

Comment:
The naming here makes it really easy to follow along with the code further down! Good job!